marker_matcher:
  ros__parameters:
    intrinsic_matrix_0: [1.625893413043423834e+03,
                         0.000000000000000000e+00,
                         6.114691731926437797e+02,
                         0.000000000000000000e+00,
                         1.627862275377948436e+03,
                         3.442905394831098533e+02,
                         0.000000000000000000e+00,
                         0.000000000000000000e+00,
                         1.000000000000000000e+00]
                            
    intrinsic_matrix_2: [1.643897500487360958e+03,
                         0.000000000000000000e+00,
                         6.331059905866444524e+02,
                         0.000000000000000000e+00,
                         1.644507446745634525e+03,
                         4.104213200244599307e+02,
                         0.000000000000000000e+00,
                         0.000000000000000000e+00,
                         1.000000000000000000e+00]
                            
    distortion_coefficients_0: [-4.651310162780882829e-01,
                                 1.155267936131824769e+00,
                                 1.167411746319183463e-03,
                                -2.916910641819991240e-03,
                                -4.800582359567718171e+00]
                                   
    distortion_coefficients_2: [-4.406292722105036863e-01,
                                 6.144566270688359699e-01,
                                -4.114959705194415080e-04,
                                 3.609129552096484832e-03,
                                -1.062926450840262271e+00]
                                   
    fundamental_matrix: [ 2.788205547579751480e-09,
                         -1.140706673629221446e-07,
                          1.251186110662436297e-04,
                         -1.055750673157773724e-07,
                          2.086353343571187610e-08,
                          7.662657768321983094e-04,
                          1.298528730735757730e-04,
                         -6.466647445016182605e-04,
                         -1.473150100904402071e-01]
                         
    fc1_weights: [ 3.580437228083610535e-02,
                   7.284557074308395386e-02,
                  -1.157589316368103027e+00,
                  -1.754364073276519775e-01,
                  -1.909652352333068848e-01,
                   3.327104151248931885e-01,
                  -5.489085912704467773e-01,
                   2.166539430618286133e-01,
                   6.200374960899353027e-01,
                  -6.508238315582275391e-01,
                   1.491757482290267944e-01,
                  -2.376191318035125732e-01,
                  -6.801597774028778076e-02,
                   2.148929238319396973e-01,
                   1.922726333141326904e-01,
                  -6.312907338142395020e-01,
                  -5.142851267009973526e-03,
                   7.479795813560485840e-02,
                   5.739377140998840332e-01,
                   1.022593259811401367e+00,
                   4.999900162220001221e-01,
                   2.040545791387557983e-01,
                  -2.607046365737915039e-01,
                   1.284240782260894775e-01]

    fc1_bias: [ 4.944512844085693359e-01,
               -2.310056984424591064e-02,
                4.441532790660858154e-01,
               -1.553014814853668213e-01,
                4.027460515499114990e-01,
               -1.911265775561332703e-02,
                9.160102345049381256e-03,
                3.033400475978851318e-01]
    
    fc2_weights: [ 4.819039702415466309e-01,
                   4.029861092567443848e-01,
                   1.914354711771011353e-01,
                  -4.866980016231536865e-01,
                  -6.428086757659912109e-01,
                  -5.930042266845703125e-02,
                  -9.468985199928283691e-01,
                   2.496316134929656982e-01,
                  -3.670258522033691406e-01,
                  -2.401712536811828613e-01,
                   2.151714116334915161e-01,
                   9.481208920478820801e-01,
                  -5.399362742900848389e-02,
                   5.239219404757022858e-03,
                   4.499680101871490479e-01,
                   6.547537446022033691e-01,
                   2.361266165971755981e-01,
                   6.065819263458251953e-01,
                  -2.679395675659179688e-01,
                   1.231537237763404846e-01,
                  -5.960191488265991211e-01,
                   9.145350009202957153e-02,
                  -1.385332584381103516e+00,
                  -1.981841921806335449e-01]
    
    fc2_bias: [ 2.578237056732177734e-01,
                1.218516528606414795e-01,
               -5.262131616473197937e-02]
    
    normalize_value_0: [1234.5, 689.5, 14.4]
    
    normalize_value_2: [1275.5, 719.5, 14.4]
    
    
marker_triangulator:
  ros__parameters:
    projection_matrix_0: [1.625893413043423834e+03,
                          0.000000000000000000e+00,
                          6.114691731926437797e+02,
                          0.000000000000000000e+00,
                          0.000000000000000000e+00,
                          1.627862275377948436e+03,
                          3.442905394831098533e+02,
                          0.000000000000000000e+00,
                          0.000000000000000000e+00,
                          0.000000000000000000e+00,
                          1.000000000000000000e+00,
                          0.000000000000000000e+00]
                             
    projection_matrix_2: [ 1.103308283516154916e+03,
                           3.828775827496751276e+02,
                           1.318839556419843802e+03,
                          -1.708313335405892758e+03,
                          -5.530340494747171078e+02,
                           1.589892071589722946e+03,
                           1.981084126184499041e+02,
                           3.389755583566757764e+02,
                          -4.733268954563312203e-01,
                          -1.624386527499366989e-02,
                           8.807370702307238952e-01,
                           3.122803150278867790e-01]

marker_tracker:
  ros__parameters:
    marker_geometry: [0.25372957696, 0.20615528128, 0.32692307692,
                      0.25372957696, 0.24095864616, 0.12833897296,
                      0.24095864616, 0.20615528128, 0.22581120481,
                      0.22581120481, 0.12833897296, 0.32692307692]
                      
    marker_geometry_angle: [1.5707963267948966, 0.6823165549405557,  0.8884797718543409,
                            0.5223843102294472, 1.2133211814340872,  1.405887161926259,
                            1.0484120165654494, 0.9127610271548704,  1.1804196098694735,
                            2.3186481892556623, 0.29193983784059957, 0.5310046264935313]

    world_rotation_matrix: [0.0, 1.0,  0.0,
                            1.0, 0.0,  0.0,
                            0.0, 0.0, -1.0]
                            
    world_translation_vector: [-0.66, -0.24, 0.0]
    
    object_rotation_matrix: [-0.63059262514554,     0.7761140000747676,  0.0,
                             -0.7761140000747676,  -0.63059262514554,    0.0,
                              0.0,                  0.0,                 1.0,
                             -0.63059262514554,     0.7761140000747676,  0.0,
                             -0.7761140000747676,  -0.63059262514554,    0.0,
                              0.0,                  0.0,                 1.0,
                             -0.63059262514554,     0.7761140000747676,  0.0,
                             -0.7761140000747676,  -0.63059262514554,    0.0,
                              0.0,                  0.0,                 1.0,
                              0.28781052365451126, -0.9576873719924034,  0.0,
                              0.9576873719924034,   0.28781052365451126, 0.0,
                              0.0,                  0.0,                 1.0]
                              
    object_translation_vector: [0.10089482002328641, 0.12417824001196281, 0.0,
                                0.10089482002328641, 0.12417824001196281, 0.0,                             
                                0.10089482002328641, 0.12417824001196281, 0.0, 
                                0.10131491864037141, 0.03741356653534816, 0.0]

/**:
  ros__parameters:
    rotation_matrix_0: [-9.659986490957758187e-01,
                         2.564896908494146421e-01,
                        -3.255224159881871249e-02,
                        -1.205959784431686332e-01,
                        -5.583614848276672316e-01,
                        -8.207856372064372685e-01,
                        -2.286989722943043590e-01,
                        -7.889521473125060691e-01,
                         5.703080652792088534e-01]
                            
    rotation_matrix_2: [-9.592521903369720659e-01,
                        -2.797037025858294945e-01,
                         4.001342391622422778e-02,
                         1.164072949397831902e-01,
                        -5.202539134917233410e-01,
                        -8.460409016007137684e-01,
                         2.574579131013920996e-01,
                        -8.069087335357847390e-01,
                         5.316142574509750141e-01]
                            
    translation_vector_0: [ 1.593791391124855794e-01,
                           -8.560620278329582333e-02,
                            2.951255116867832484e+00]
                               
    translation_vector_2: [ 3.237276846596536806e-01,
                           -2.828835220492980640e-01,
                            2.837666216447339629e+00]

